### PR TITLE
fix(TBD-12813): derby upgraded to 10.12.1.1, json-smart to 2.4.7

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by lbourgeois — APPROVED** [review]
Some old plugins from old distros not even displayed anymore in the Studio could have been removed instead of updating them.

**Review by AlixMetivier — DISMISSED** [review]
those plugins will be deprecated / removed, lgtm but might not be needed ?

**Review by cbadillo1603 — DISMISSED** [review]
reviewed

File: main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml [hunk 1]
```diff
@@ -651,7 +657,11 @@
       <!-- EMR 4.5.0 ClassLoaders -->
       <classloader
             index="HIVE2:AMAZON_EMR:EMR_4_5_0:STANDALONE"
+<<<<<<< HEAD
```
Comment (nrousseau) [inline main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:660]: conflict to resolve

**Review by nrousseau — CHANGES_REQUESTED** [review]

**Review by nrousseau — APPROVED** [review]

**Review by firefryy — APPROVED** [review]
lgtm

**Review by firefryy — COMMENTED** [review]
lgtm

**Pull request merged**